model/clusters_mgmt/v1/add_on_type.model

```diff
@@ -36,4 +36,10 @@ class AddOn {
 
 	// Used to determine how many units of quota an add-on consumes per resource name.
 	ResourceCost Float
+
+	// The namespace name in which the addon is installed.
+    TargetNamespace String
```

Comment:
Why is this needed?

Reply:
In the documentation comments, you mean in the .model file itself?
This field is required sine we want to present the consumer with the information of which namespace the addon is installed in. We want to show this the targetNamespace on the addon tab.

Reply:
Yes, I mean in the documentation comments in the .model file.